style.css

```diff
@@ -0,0 +1,162 @@
+html,
+body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    font-size: 100%;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    background: linear-gradient(180deg, rgba(255, 218, 185, 1) 0%, rgba(248, 173, 157, 1) 100%);
+    transition: background-color 2s ease;
+}
+
+header {
+    height: 33.33%;
+    position: relative;
+}
+
+main {
+    height: 66.66%;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+}
+
+h1 {
+    text-align: center;
+    position: absolute;
+    top: 20%;
+    right: 50%;
+    transform: translateX(50%);
+    font-size: 5em;
+    border-bottom: 5px solid #f08080;
+    border-image: linear-gradient(90deg, rgb(252, 129, 129) 0%, rgb(252, 205, 179) 100%);
+    border-image-slice: 1;
+}
+
+.secret {
+    height: 5rem;
+    width: 5rem;
+    text-align: center;
+    font-size: 4em;
+    position: absolute;
+    top: 90%;
+    right: 50%;
+    transform: translateX(50%);
+    background-color: #f8ad9d;
+    border-radius: 20px;
+}
+
+.range {
+    position: absolute;
+    top: 2rem;
+    left: 85%;
+    text-align: center;
+    font-size: larger;
+}
+
+
+
+.left {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 2em;
+}
+
+.user--input:focus {
+    outline: none;
+}
+
+.user--input {
+    font-family: 'Roboto', sans-serif;
+    background: none;
+    width: 15rem;
+    height: 8rem;
+    font-size: 5rem;
+    border: 12px solid #f4978e;
+    border-radius: 10px;
+    text-align: center;
+
+
+}
+
+.btn:hover {
+    background-color: #f8ad9d;
+}
+
+.btn:active {
+    background-color: #f4978e;
+}
+
+.btn {
+    font-family: 'Roboto', sans-serif;
+    background-color: #fbc4ab;
+    border: none;
+    border-radius: 10px;
+    cursor: pointer;
+}
+
```

Comment:
You UI could be better with this some style of the disabled button:

```css
.btn--guess:disabled{
    cursor: not-allowed;
    pointer-events: none;
}
```

Like so, the user really understand it's not allowed to click.